topic-operator/src/main/java/io/strimzi/operator/topic/TopicConfigsWatcher.java

```diff
@@ -11,18 +11,16 @@
  */
 class TopicConfigsWatcher extends ZkWatcher {
 
-    private static final String CONFIGS_ZNODE = "/config/topics";
-
     TopicConfigsWatcher(TopicOperator topicOperator) {
         super(topicOperator, CONFIGS_ZNODE);
```

Comment:
Do we really need to take `CONFIGS_ZNODE` defined in `ZkWatcher` and pass it through `TopicConfigsWatcher` into `ZkWatcher`? Maybe it has some reasons ... but it makes me wonder if it should be a parameter in the constructor.

Reply:
IDK. I just removed unused property as it had the same value as in it has in its super class,

Reply:
Ok.